data/strings/types_strings.txt

```diff
@@ -9193,6 +9193,318 @@
+    da = Audiolog
+    de = Audiologe
+    el = Ακουολόγος
+    es = Audiólogo
+    et = Audioloog
+    eu = Audiologoa
+    fa = شنوایی شناس
+    fi = Audiologi
+    fr = Audiologiste
+    he = אודיולוג
+    hu = Audiológus
+    id = Audiologis
+    it = Audiologo
+    ja = オーディオロジスト
+    ko = 청력학자
+    mr = ऑडिओलॉजिस्ट
+    nb = Audiolog
+    nl = Audioloog
+    pl = Audiolog
+    pt = Audiologista
+    pt-BR = Audiologista
+    ro = Audiolog
+    ru = Аудиолог
+    sk = Audiológ
+    sv = Audiolog
+    sw = Mtaalamu wa kusikia
+    th = นักโสตสัมผัสวิทยา
+    tr = Odyolog
+    uk = Аудіолог
+    vi = chuyên gia thính học
+    zh-Hans = 听力学家
+    zh-Hant = 聽力學家
+
+  [type.healthcare.blood_donation]
+    en = Blood donation
+    ar = تبرع بالدم
+    be = Донарства крыві
+    bg = Даряване на кръв
+    cs = Darování krve
+    da = Bloddonation
+    de = Blutspende
+    el = Αιμοδοσία
+    es = Donación de sangre
+    et = Veredoonorlus
+    eu = Odol ematea
+    fa = اهدای خون
+    fi = Verenluovutus
+    fr = Don du sang
+    he = תרומת דם
+    hu = Véradás
+    id = Donor darah
+    it = Donazione di sangue
+    ja = 献血
+    ko = 헌혈
+    mr = रक्तदान
+    nb = Bloddonasjon
+    nl = Bloeddonatie
+    pl = Oddawanie krwi
+    pt = Doação de sangue
+    pt-BR = Doação de sangue
+    ro = Donarea de sânge
+    ru = Донорство крови
+    sk = Darovanie krvi
+    sv = Blodgivning
+    sw = Utoaji wa damu
+    th = การบริจาคเลือด
+    tr = Kan Bağışı
+    uk = Донорство крові
+    vi = Hiến máu
+    zh-Hans = 献血
+    zh-Hant = 獻血
+
+  [type.healthcare.optometrist]
+    en = Optometrist
+    ar = طبيب العيون
+    be = Акуліст
+    bg = Оптометрист
+    cs = Optometrista
+    da = Optometrist
+    de = Augenoptiker
+    el = Οπτομέτρης
+    es = Optometrista
+    et = Optometrist
+    eu = Optometrista
+    fa = اپتومتریست
+    fi = Optometristi
+    fr = Optométriste
+    he = אופטומטריסט
+    hu = Optometrista
+    id = Ahli kacamata
+    it = Optometrista
+    ja = オプトメトリスト
+    ko = 검안사
+    mr = ऑप्टोमेट्रिस्ट
+    nb = Optometrist
+    nl = Optometrist
+    pl = Optometrysta
+    pt = Optometrista
+    pt-BR = Optometrista
+    ro = Optometrist
+    ru = Оптометрист
+    sk = Optometrista
+    sv = Optometrist
+    sw = Daktari wa macho
+    th = นักทัศนมาตร
+    tr = Optometrist
+    uk = Лікар-оптометрист
+    vi = bác sĩ nhãn khoa
+    zh-Hans = 验光师
+    zh-Hant = 驗光師
+
+  [type.healthcare.podiatrist]
+    en = Podiatrist
+    ar = طبيب الأرجل
+    be = Ортопед
+    bg = Подиатрист
+    cs = Podiatr
+    da = Fodterapeut
+    de = Podologe
+    el = Ποδίατρος
+    es = Podólogo
+    et = Jalaravi
+    eu = Podologoa
+    fa = متخصص پا
+    fi = Jalkaterapeutti
+    fr = Podologue
+    he = פודיאטר
+    hu = Podiatrist
+    id = Ahli penyakit kaki
+    it = Podologo
+    ja = 足病医
+    ko = 발 전문의
+    mr = पोडियाट्रिस्ट
+    nb = Fotterapeut
+    nl = Podotherapeut
+    pl = Podolog
+    pt = Podiatrista
+    pt-BR = Podiatrista
```

Comment:
pt = Podologista
pt-BR = Podólogo

Reply:
Thanks!
Done in 5d32dd98de7b8591e4719e2842b617d50026572f